Source/Scene/Cesium3DTileset.js

```diff
@@ -2221,7 +2221,7 @@ define([
     Cesium3DTileset.prototype.destroy = function() {
         // Destroy debug labels
         if (defined(this._tileInfoLabels)) {
-            this._tileInfoLabels.destroy();
+            this._tileInfoLabels = this._tileInfoLabels && this._tileInfoLabels.destroy();
         }
```

Comment:
Remove the `if (defined(this._tileInfoLabels)) {` wrapper, it is unnecessary now.